src/com/epam/Chat.java

```diff
@@ -0,0 +1,51 @@
+package com.epam.training;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.util.List;
+
+public class Chat {
+    public static Scanner input = new Scanner(System.in);
+    public static List<Member> members;
+    static String history = "";
+
+    public static void setMembers(int memCount) {
+        members = new ArrayList<Member>(0) {
+        };
+        Member cur;
+        for (int i = 1; i <= memCount; i++) {
+            cur = new Member("member" + i);
+            members.add(i - 1, cur);
+        }
+    }
+
+
+    public static void startChat(int memCount) {
+        setMembers(memCount);
+        Member cur;
+        while (true) {
```

Comment:
what if all members were removed from chat? it will go with while(true) and the program will not stop at all. change stop break the while when memCount==0